tests/gdimageellipse/bug00169.c

```diff
@@ -0,0 +1,22 @@
+/**
+ * add test case for gdImageEllipse
+ */
+
+#include "gd.h"
+#include "gdtest.h"
+
+int main()
+{
+	gdImagePtr im;
+	int white = 0;
+
+	im = gdImageCreateTrueColor(2200, 2200);
+	white = gdImageColorAllocate(im, 255, 255, 255);
+
+	gdImageEllipse(im, 1100, 1100, 2200, 2200, white);
+  gdAssertImageEqualsToFile("gdimageellipse/bug00169_exp.png", im);
```

Comment:
Nit: indentation (also in the other test below)

Reply:
Fixed!

Sorry I used a new computer with vim config auto expanding tab to spaces.

Reply:
Thanks for your careful review :)